webdriver-ts/src/benchmarks.ts

```diff
@@ -380,9 +380,9 @@ class BenchStartup extends Benchmark {
             type: BenchmarkType.STARTUP,
         })
     }
-    async init(driver: WebDriver) { await driver.get(`http://localhost:8080/`); }
-    async run(driver: WebDriver, framework: FrameworkData) {
-        await driver.get(`http://localhost:8080/${framework.uri}/`);
+    async init(driver: WebDriver, framework: FrameworkData, port: number) { await driver.get(`http://localhost:` + port + `/`); }
```

Comment:
Had to add the `FrameworkData` parameter from the base method to fix an overload incompatibility.
```
src/benchmarks.ts(374,7): error TS2415: Class 'BenchStartup' incorrectly extends base class 'Benchmark'.
  Types of property 'init' are incompatible.
    Type '(driver: WebDriver, port: number) => Promise<void>' is not assignable to type '(driver: WebDriver, framework: FrameworkData, port: any) => Promise<any>'.
      Types of parameters 'port' and 'framework' are incompatible.
        Type 'FrameworkData' is not assignable to type 'number'.
```